url_shortener/urls/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from .models import Urls
+
+class NewUrlForm(forms.ModelForm):
```

Comment:
consider forms.URLField(), a built in form class from django
We tried this but didn't impact the form input not being displayed, not sure what's causing the url input section to be hidden.